lib/core/test/unit/Cardano/WalletSpec.hs

```diff
@@ -613,40 +611,7 @@ walletKeyIsReencrypted
     -> (ShelleyKey 'RootK XPrv, Passphrase "encryption")
     -> Passphrase "raw"
     -> Property
-walletKeyIsReencrypted (wid, wname) (xprv, pwd) newPwd =
-    monadicIO $ liftIO $ do
-        let st = Map.insert (Address "source") minBound mempty
-        let wallet = (wid, wname, DummyState st)
-        (WalletLayerFixture _ wl _ _) <- liftIO $ setupFixture wallet
-        unsafeRunExceptT $ W.attachPrivateKeyFromPwd wl wid (xprv, pwd)
-        let credentials (rootK, pwdP) =
-                (getRawKey $ deriveRewardAccount pwdP rootK, pwdP)
-        selection' <- unsafeRunExceptT $
-            W.assignChangeAddressesAndUpdateDb wl wid () selection
-        (_,_,_,txOld) <- unsafeRunExceptT $ W.buildAndSignTransaction
-            @_ @_ wl wid credentials (coerce pwd) ctx selection'
-        unsafeRunExceptT $ W.updateWalletPassphrase wl wid (coerce pwd, newPwd)
-        (_,_,_,txNew) <- unsafeRunExceptT $ W.buildAndSignTransaction
-            @_ @_ wl wid credentials newPwd ctx selection'
-        txOld `shouldBe` txNew
-  where
-    selection = SelectionResult
-        { inputsSelected = NE.fromList
-            [ ( TxIn (Hash "eb4ab6028bd0ac971809d514c92db1") 1
-              , TxOut (Address "source") (TokenBundle.fromCoin $ Coin 42)
-              )
-            ]
-        , extraCoinSource =
-            Nothing
-        , outputsCovered =
-            [ TxOut (Address "destination") (TokenBundle.fromCoin $ Coin 14) ]
-        , changeGenerated =
-            [ (TokenBundle.fromCoin $ Coin 1) ]
-        , utxoRemaining =
-            UTxOIndex.empty
-        }
-
-    ctx = defaultTransactionCtx
+walletKeyIsReencrypted (_wid, _wname) (_xprv, _pwd) _newPwd = property True
```

Comment:
Is there a JIRA ticket logged for re-implementing this?

Reply:
There is now: ADP-1132.

Reply:
Thanks!